src/main/kotlin/me/bristermitten/devdenbot/data/MessageCache.kt

```diff
@@ -0,0 +1,37 @@
+package me.bristermitten.devdenbot.data
+
+import java.util.*
+import kotlin.collections.HashMap
+
+class MessageCache {
```

Comment:
If you're only using the class for the companion object, the class itself can become an `object`